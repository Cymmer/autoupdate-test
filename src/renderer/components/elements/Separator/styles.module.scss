@import '../../../scss/variables';
@import '../../../scss/themes';

.Separator {
  border: none;
  border-top: 1px solid;
  @include theme {
    border-color: t('borderColor');
  }
  margin: 0;
  width: 100%;
}
