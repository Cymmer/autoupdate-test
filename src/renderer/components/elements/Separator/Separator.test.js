import React from 'react';
import { shallow } from 'enzyme';
import Separator from './index';

describe('RENDERING - Separator', () => {
  it('should render with default props', () => {
    const component = shallow(<Separator />);
    expect(component).toMatchSnapshot();
  });
});
