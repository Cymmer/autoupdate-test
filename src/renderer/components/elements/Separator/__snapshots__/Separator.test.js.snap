// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RENDERING - Separator should render with default props 1`] = `
<hr
  className="Separator"
/>
`;
